### DISCUSSION TITLE
[v3] Problem with compiling _meta.js files while uploading to Github Pages

### DISCUSSION THREAD

**Discussion by kpietak** [shuding/nextra#3735] [Q&A]
Hi,

I've upgraded my site to Nextra v3 and it works pretty well, but when I publish it on Github Pages I've got the following error during the build:

```
Build optimization failed: found pages without a React Component as default export in 
pages/_meta
pages/topics/01-intro/_meta
...
```

All my `_meta.js` contain `export default` clause but somehow the builder does not recognise them.

The whole error message you can see at https://github.com/kpietak/rust-course/actions/runs/11997724944/job/33443671115

Any ideas what could be wrong? I've tried many various configuration according to migration guide, but everything works locally, nothing on GH Pages.

One of my `_meta.js` files is listed below:
```
export default {
  "index": "Wprowadzenie",
  "02-move": "Przenoszenie własności",
  "03-borrow": "Referencje i pożyczki",
  "04-slice": "Wycinki",
  "05-exercises": "Ćwiczenia"
}
```

Any help will be appreciated. 

Ps. Exactly the same configuration compiles properly in Vercel so I suppose this is something related to the GH environment.

**Comment by codenamethanos** [comment]
I am facing the same issue during build when deploying in Vercel. How did you make it work in Vercel?

**Comment by kpietak** [comment]
I did nothing, the same code doesn't build on Github Pages, but it works well in Vercel. If you can send me a link to your repo I can try to find differences in configuration.
